scan/cli.go

```diff
@@ -177,6 +184,10 @@ func AuditCmd(c *cli.Context) error {
 			err = AuditPipCmd(c)
 		case coreutils.Pipenv:
 			err = AuditPipenvCmd(c)
+		case coreutils.Dotnet:
+			break
```

Comment:
Shouldn't we have ```fallthrough``` here instead of  ```break```? 
Because ```fallthrough``` will make the outcome of a  **Dotnet** detection similar to ** Nuget**.

Reply:
But currently, we are identifying Dotnet and NuGet in the exact same way (the existence of .sln file).
If we change the `break` to `fallthrough` the audit command will always run twice.

Reply:
Got it. NP.